collect_app/src/main/java/org/odk/collect/android/widgets/RankingWidget.java

```diff
@@ -98,7 +98,7 @@ public void onButtonClick(int buttonId) {
         if (formController != null) {
             formController.setIndexWaitingForData(getFormEntryPrompt().getIndex());
         }
-        RankingWidgetDialog rankingWidgetDialog = RankingWidgetDialog.newInstance(savedItems == null ? items : savedItems, getFormEntryPrompt().getIndex());
+        RankingWidgetDialog rankingWidgetDialog = new RankingWidgetDialog(savedItems == null ? items : savedItems, getFormEntryPrompt().getIndex());
         rankingWidgetDialog.show(((FormEntryActivity) getContext()).getSupportFragmentManager(), "RankingDialog");
```

Comment:
This is not really related to this PR but it just came to mind that I don't really know when `DialotUtils.showIfNotShowing` makes a difference. Should it be used here?

Reply:
Can't be sure whether it's needed here I doubt but... generally it's good to use that method it's a defensive approach which could prevent some strange and rare crashes but that method uses bundle to pass data so that's what I need to avoid in this case.